internal/librarian/generate.go

```diff
@@ -209,6 +211,17 @@ func (r *generateRunner) runGenerateCommand(ctx context.Context, outputDir strin
 		return libraryID, r.containerClient.Generate(ctx, generateRequest)
 	}
 	slog.Info("No matching library found (or no repo specified)", "path", r.cfg.API)
+	// Handle push config flag
+	if r.cfg.PushConfig != "" {
+		if r.cfg.GitHubToken == "" {
+			return "", fmt.Errorf("GitHub token is required to push config")
```

Comment:
I think it's better to do this check earlier in the flow and fail early. 
Maybe in newGenerateRunner()?